### ISSUE TITLE
Build turbine-web app

### ISSUE THREAD

**Issue by karthik101** [Netflix/Turbine#97]
I have been trying to build turbine-web war file as the shipped turbine-web-1.0.0 does not seem to work for me and decided to compile it from source but cannot understand how to start.

" ./gradlew build " only builds but doesn't create any war file. If any of you guys could share on how to create turbine web war file and successfully deploy it.

**Comment by mohan-mishra** [comment]
@karthik101  were you able to do this?

**Comment by maxwelltan** [comment]
why i can't find the turbine-web module in the Turbine project, only has turbine-core and turbine-ext. 
are the wiki of https://github.com/Netflix/Hystrix/wiki/Dashboard#build-1 deprecated.

**Comment by karthik101** [comment]
@mohan-mishra No, I used the shipped one by netflix and It worked after i removed one file ( Causing tomcat/jboss launch issues - Its been a year i don't remember correctly) using VIM editor.
